src/utils/validators.ts

```diff
@@ -33,9 +33,9 @@ export function validateChannel(channel: number, image: Image): void {
  * @param image - Image from which the value comes.
  */
 export function validateValue(value: number, image: Image): void {
```

Comment:
I think `validateValue` is used in other places where we really want the value to be an integer?

Reply:
It's apparently only used in `fill` and `fillAlpha` when the user passes data to be put directly in the image. Since `data` is a typed array, shouldn't it clamp the values automatically?

Reply:
I added a test and the values are indeed clamped. I don't know then if we want to force the user to input integers?

Reply:
ok